src/ontodia/workspace/workspace.ts

```diff
@@ -22,6 +22,12 @@ export interface Props {
     isViewOnly?: boolean;
     isDiagramSaved?: boolean;
     hideTutorial?: boolean;
+    linkStyles?: {
```

Comment:
It will be better to rebase this commit on top of ["element color" pull request ](https://github.com/ontodia-org/ontodia/pull/5) and define `linkStyles` inside `ViewOptions` for DiagramView.